drivers/include/net/netdev2/lorawan.h

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (C) Fundación Inria Chile 2017
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @ingroup     drivers_netdev_netdev2
+ * @{ *
+ * @file
+ * @brief       Definitions for netdev2 common lorawan code
+ *
+ * @author      José Ignacio Alamos <jose.alamos@inria.cl>
+ */
+
+#ifndef NETDEV2_ETH_H
+#define NETDEV2_ETH_H
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "net/netdev2.h"
+#include "net/netopt.h"
+#include "net/gnrc/lorawan/timer.h"
+#include "net/gnrc/lorawan/definitions.h"
+#include "net/gnrc/pkt.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define KEY_SIZE		16	
```

Comment:
indent is all off (here and below). Please use type specifiers (```(16U```) where they might make a difference.